k8s-alpha.yml

```diff
@@ -217,3 +217,103 @@ spec:
 
 # END Mathoid
 
+---
+# Begin ChangeProp
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: kafka-deploy
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        service_name: kafka
+    spec:
+      containers:
+      - name: kafka
+        image: wurstmeister/kafka:0.9.0.1-1
+        ports:
+        - containerPort: 9092
+        env:
+          - name: KAFKA_BROKER_ID
+            value: '1'
+          - name: KAFKA_ADVERTISED_HOST_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
```

Comment:
Yup. That's the idea - the `kafka-svc` is used as a bootstrap node, and if we have many replicas, it will load-balance. When the client is connected it needs to get the exact IP of each pod so that the client can send request to the partition leader directly.